python/lsst/meas/algorithms/detection.py

```diff
@@ -166,154 +234,154 @@ def setDefaults(self):
-@code
-examples/measAlgTasks.py --ds9
-@endcode
-@dontinclude measAlgTasks.py
-The example also runs the SourceMeasurementTask; see @ref meas_algorithms_measurement_Example for more
-explanation.
+        # (You may not be happy that the threshold was set in the config before creating
+        #  the Task rather than being set separately for each exposure. You can reset it 
+        #  just before calling the run method if you must, but we should really implement a better solution).
+        # We can then unpack and use the results:
 
-Import the task (there are some other standard imports; read the file if you're confused)
-@skipline SourceDetectionTask
+        sources = result.sources
+        print("Found %d sources (%d +ve, %d -ve)" % (len(sources), result.fpSets.numPos, result.fpSets.numNeg))
 
-We need to create our task before processing any data as the task constructor
-can add an extra column to the schema, but first we need an almost-empty Schema
-@skipline makeMinimalSchema
-after which we can call the constructor:
-@skip SourceDetectionTask.ConfigClass
-@until detectionTask
+        #To investigate the Debug variables, put something like
 
-We're now ready to process the data (we could loop over multiple exposures/catalogues using the same
-task objects).  First create the output table:
-@skipline afwTable
+        import lsstDebug
+        def DebugInfo(name):
+            di = lsstDebug.getInfo(name)        # N.b. lsstDebug.Info(name) would call us recursively
+            if name == "lsst.meas.algorithms.detection":
+                di.display = 1
+            return di
+        lsstDebug.Info = DebugInfo
 
-And process the image
-@skipline result
-(You may not be happy that the threshold was set in the config before creating the Task rather than being set
-separately for each exposure.  You @em can reset it just before calling the run method if you must, but we
-should really implement a better solution).
+        #into your debug.py file and run measAlgTasks.py with the –debug flag.
 
-We can then unpack and use the results:
-@skip sources
-@until print
-
-<HR>
-To investigate the @ref meas_algorithms_detection_Debug, put something like
-@code{.py}
-    import lsstDebug
-    def DebugInfo(name):
-        di = lsstDebug.getInfo(name)        # N.b. lsstDebug.Info(name) would call us recursively
-        if name == "lsst.meas.algorithms.detection":
-            di.display = 1
-
-        return di
-
-    lsstDebug.Info = DebugInfo
-@endcode
-into your debug.py file and run measAlgTasks.py with the @c --debug flag.
     """
+
     ConfigClass = SourceDetectionConfig
     _DefaultName = "sourceDetection"
 
     def __init__(self, schema=None, **kwds):
-        """!Create the detection task.  Most arguments are simply passed onto pipe.base.Task.
+         """Create the detection task.  Most arguments are simply passed onto `pipe.base.Task`.
```

Comment:
Remember that we don't put docstrings on our `__init__` methods. Instead, the parameters go into the class docstring.